### DISCUSSION TITLE
Do Bots Need AI Tweaks for Mechtrauma?

### DISCUSSION THREAD

**Discussion by MapleWheels** [PeasantFoundry/Mechtrauma#41] [Polls]
This would be a further off goal but I think that AI action behaviour towards certain Mechtrauma machines needs to be addressed. Bots have a default behaviour that they will apply even to modded machines based on certain assumptions for actions like [Repair]. We will need to make sure that bots don't accidentally start breaking Mechtrauma subs with incorrect behaviour.

**Comment by PeasantFoundry** [comment]
Bots are going to be problematic. However, a lot of the impacts can be mitigated by updating the xml tagging to tell bots what goes where. I believe that you can get them to treat broken items in mechtrauma like oxygentanks. For example fuses, broken fuses want to be in the deconstructor or some cabinet and good fuses want to be in electrical panels. I've see it work but haven't bothered with it myself.

One of the biggest dangers with bots is having them hurt themselves by repairing items when they are one. I've already reduced the penalties for bots failing a repair check vs a player. There could also be a config for "BOT MODE" that simply does not allow dangerous items to be repaired while they are on.
